apps/mobile/mobile-app/types/index.ts

```
@@ -0,0 +1,13 @@
+export interface Item {
+    id: string;
+    title: string;
+    imageUrl: string;
+    category: string;
+    image: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)